index.html

```diff
@@ -168,7 +172,46 @@ <h3>Skills</h3>
                 </ul>
             </li>
         </ul>
+        </div>
+<div class="form-container">
+    <h4 class="aboutme-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+    <form action="https://formspree.io/f/xrgvbqgv" method="POST">
+        <input type="text" name="fullname" placeholder="Full Name" maxlength="30" required>
+        <input type="email" name="email" placeholder="Email Address" required>
+        <textarea type="text" name="textarea" maxlength="500" required>Enter text here</textarea>
+        <button type="submit">Get in touch</button>
+    </form>
+</div>
+<footer>
+    <div class="contacts">
+        <ul class="icons2">
+            <li>
+                <a href="https://twitter.com/Stephen57913145"><i class="fa-brands fa-twitter"></i></a>
+            </li>
+            <li>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/stephen-katuli-a92752251/"><i class="fa-brands fa-linkedin"></i></a>
+            </li>
+            <li>
+            </li>
+            <li>
+                <a href=""><img src="/images/iconmonstr-medium-1.svg"></a>
```

Comment:
- [ ] One of the objectives of this project is to understand how to parse a design template precisely. Let's try to stick to the design as much as possible. 🙂️
Kindly add a `.` before your image paths so they can get displayed correctly on all platforms. 🤓️
Please add the background image on the right side to match the design template. 😎️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/213101975-4a5f2c0c-f624-4e34-bbf0-670adb41e2f9.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/213101972-1bdc9bfb-9aa3-44bd-a3be-9284d025e379.png" width="300">|